Clarify service name in Discord (#22363)
home-assistant/home-assistant.github.io: source/_integrations/discord.markdown

{% include integrations/config_flow.md %}

<div class='note'>

> Note that the name you give your application in the [Discord My Apps page](https://discordapp.com/developers/applications/me) will determine the name of the notify service. For example: if you enter "Discord", the service will be named notify.discord.

Reviewer: Small tweak, also a small change to make it different from the default used in the examples on this page.

```suggestion
The name you give your application on the [Discord My Apps page](https://discordapp.com/developers/applications/me) will determine the name of the notify service. For example: if you enter "Discord Chat", the service will be named `notify.discord_chat`.
```
